core/platform/models.py

```diff
@@ -25,8 +25,15 @@
 import feconf
 import python_utils
 
+from types import ModuleType # isort:skip # pylint: disable=import-only-modules,unused-import
+from typing import Text, List, Tuple, Optional, Type # isort:skip # pylint: disable=unused-import
+
+MYPY = False
+if MYPY:
+    from core.storage.base_model.gae_models import BaseModel # pragma: no cover # pylint: disable=import-only-modules,unused-import
```

Comment:
What's MYPY here? is this a constant to enable/disable the mypy checks?

Reply:
I'll keep this in mind. Thanks!!